app/Http/Controllers/Controller.php

```diff
@@ -13,7 +13,11 @@ class Controller extends BaseController
 
     public function index()
     {
-
+        /**
+         * Todo
+         * - Cache the response of recent and popular articles
+         * - Select only the fields we need from the database, in order to make the home page render as fast as possible
+         */
```

Comment:
In the home page endpoint, we return the view with two lists of recent articles and popular articles. These queries are executed every time someone visits the home page, therefore prolonging the load time.
Maybe there are ways to load those articles faster?

**Hints**:

- Maybe there is a way to persist the query result to be reused later within a specific time frame?
- These queries retrieve all columns from the articles table. Consider selecting the required fields only.